src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/db/StatusConfig.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.sandl.snlevents.model.db;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+public class StatusConfig implements Serializable {
+
+    @Id
+    private String status;
```

Comment:
Having second thoughts whether this should be string or an enum of statuses saved as string...